tests/conf_test_aktualizr_info.toml

```diff
@@ -0,0 +1,25 @@
+[core]
+polling = true
+polling_sec = 10
+
+
+[ostree]
+os = "myos"
+
+[tls]
+certificates_directory = "test_data/prov/certs"
+
+[uptane]
+metadata_path = "test_data/prov/repo"
+disable_keyid_validation = true
+primary_ecu_hardware_id = "master"
+key_source = "pkcs11"
+public_key_path="01"
+private_key_path = "01"
+
+[p11]
+module = "/usr/lib/softhsm/libsofthsm2.so"
+pass = "123456"
+
+[provision]
+provision_path = "/home/patriotyk/Downloads/credentials(4).zip"
```

Comment:
I think this is causing the following test failures

{code}

The following tests FAILED:
	 48 - aktualizr-info-images-root (Failed)
	 49 - aktualizr-info-director-root (Failed)
	 50 - aktualizr-info-director-target (Failed)
	 51 - aktualizr-info-images-target (Failed)
{code}

Reply:
Weird, those work on travis but not on my machine

Reply:
I don't think so, but I have removed those lines. Could You run ctest -V -R aktualizr-info and give me output?